### ISSUE TITLE
Google AI Princeton: Current and Future Research

### ISSUE THREAD

**Issue by ikedaosushi** [ikedaosushi/tech-news#9545]
Google AI Princeton: Current and Future Research<br>
Gradient descent (GD), and its randomized version, stochastic gradient descent (SGD), are the methods of choice for optimizing the weights of neural networks.<br>
https://ift.tt/2rInMos
